### ISSUE TITLE
iOS: (Error: this._getLatestWebviewContextForTitle is not a function) on switching to hybrid app webview 

### ISSUE THREAD

**Issue by mayureshshirodkar** [appium/appium#13063]
## The problem

Appium script is unable to switch to webview and fails with this error.

I have seen this happening with:
Appium version : Appium v1.14.0-beta.2
Appium xcuitest version: appium-xcuitest 2.126.2

Works fine after downgrading appium-xcuitest:
Appium version : Appium v1.14.0-beta.2
Appium xcuitest version: appium-xcuitest v2.123.0

**Error:**
```
[XCUITest] Error: Could not navigate to webview! Err: this._getLatestWebviewContextForTitle is not a function
```
## Environment

* Appium version (or git revision) that exhibits the issue: v1.14.0-beta.2
* Last Appium version that did not exhibit the issue (if applicable): 
* Desktop OS/version used to run Appium: 10.14.5 Mac OS Mojave
* Node.js version (unless using Appium.app|exe): v10.16.2
* Npm or Yarn package manager: v6.9.0
* Mobile platform/version under test: iOS 12.4
* Real device or emulator/simulator: iPhone X

## Details

On installing the script, i have enabled autowebview which automatically switches to  the webviews available. 

## Link to Appium logs

Error Gist: https://gist.github.com/mayureshshirodkar/57a0542a3a521dbf49e47761972d3e35

**Comment by mykola-mokhnach** [comment]
@imurchie FYI

**Comment by imurchie** [comment]
Hmm, yes. This got messed up in some refactoring. I'll fix...

**Comment by mayureshshirodkar** [comment]
@imurchie when will this fix available in the beta?

**Comment by umutuzgur** [comment]
@mayureshshirodkar You should be able to use it if you reinstall beta right now

**Comment by mayureshshirodkar** [comment]
Thanks @umutuzgur  @imurchie @mykola-mokhnach 
The fix is on appium-xcuitest-driver@v2.126.4 and working.
